checkstyle-sonar-plugin/src/main/java/org/sonar/plugins/checkstyle/CheckstyleRulesDefinition.java

```diff
@@ -60,8 +64,38 @@ static void extractRulesData(NewRepository repository, String xmlRulesFilePath,
         ExternalDescriptionLoader.loadHtmlDescriptions(repository, htmlDescriptionFolder);
         try (InputStream resource = CheckstyleRulesDefinition.class
                 .getResourceAsStream("/org/sonar/l10n/checkstyle.properties")) {
-            PropertyFileLoader.loadNames(repository, resource);
+            loadNames(repository, resource);
         }
         SqaleXmlLoader.load(repository, "/com/sonar/sqale/checkstyle-model.xml");
     }
+
+    private static void loadNames(NewRepository repository, InputStream stream) {
+        // code taken from https://github.com/SonarSource/sslr-squid-bridge/blob/2.5.2/
+        // src/main/java/org/sonar/squidbridge/rules/PropertyFileLoader.java#L46
```

Comment:
please exlpain why sslr-squid-bridge 2.5.2 is used while in pom dependency we use 2.7.0.377

FYI: you can start line comment with `http` our checkstyle config ignore lines with URLs.

Reply:
This method got removed after version 2.5.2 but we need 2.7.0.377 for SQ compatibility.
Hence, I have migrated the "old" code from sslr-squid-bridge here and referenced with the last available release.

Reply:
Starting line comment with `http` didn't solve the line length issue. I always get line too long.